web/public/locales/de.json

```diff
@@ -1420,10 +1423,6 @@
       "countryName": "Großbritannien",
       "zoneName": "Shetland Islands"
     },
-    "GB-ZET": {
-      "countryName": "Großbritannien",
-      "zoneName": "Shetland Islands"
-    },
```

Comment:
Apparently there was a duplicate here, removed it when I fixed the merge conflicts.